runtime/Java/src/org/antlr/v4/runtime/atn/ATNDataWriter.java

```diff
@@ -0,0 +1,36 @@
+package org.antlr.v4.runtime.atn;
```

Comment:
This is another file that's in one target which will diverge from other targets and then new targets will have yet another file to implement. A lotta these files are really related to the way Java likes to do things.

Reply:
`ATNDataWriter` is only actual for Java, maybe C++ because other targets don't write they only read ATN.